ChangeLog

```diff
@@ -1,3 +1,10 @@
+2015-01-13  Daniel Standage <daniel.standage@gmail.com>
+
+    * tests/khmer_tst_utils.py, tests/test_sandbox_scripts.py: removed
+    unused module imports
+    * .gitignore: added pylint_report.txt so that it is not accidentally
+    committed after running make diff_pylint_report
```

Comment:
Thanks! Do you know about `git add -p`? I use that instead of `git commit -a` to avoid that problem. The `.gitignore` update is still useful.